.github/workflows/changelog-label.yml

```diff
@@ -0,0 +1,20 @@
+name: Require changelog entry
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
nit: why do we need `unlabeled` here 🤔

Reply:
I'm not certain, but I think without `unlabeled` the check wouldn't re-run when the `changelog` label is removed. Probably won't happen often but a reviewer might change their mind and remove the label which should unblock the merge